add new key mappings to the Traditional Mongolian Keyboard layout

Satsrag opened the pull request:
1. Map `v+4` to input FVS4.
2. Add an alternative mapping for MVS using `v+space`, since the JIS Japanese keyboard does not have the "·" key (to the left of 1).
![IMG_1856](https://github.com/user-attachments/assets/621185c2-0871-49a0-bcd2-149c022c91b8)

suragch commented:
Does this change just add mappings? It doesn't break any of the current mappings?

Satsrag committed:
add new key mappings

- Map v+4 to FVS4
- Map v+space to MVS (for JIS keyboards without the "·" key (left of 1))

Satsrag commented:
Sorry for the late reply — we were on holiday. I’ve removed some unnecessary code, and now only two mappings remain: new mvs and fvs4.

suragch commented:
I'm merging this but I also opened issue #3 to update the documentation and downloadable files.

Thank you for your update!
